languages.yaml

```diff
@@ -15,23 +16,25 @@ en:
 
 es:
   PLUGIN_MAINTENANCE:
-    OFFLINE_MARKDOWN: "# Sitio offline\n## Por favor, inténtalo más tarde..."
-    AUTHORIZED_ACCESS: "**Se requiere acceso autorizado**"
+    OFFLINE_H1: Sitio offline
+    OFFLINE_H2: Por favor inténtalo mas tarde...
+    AUTHORIZED_ACCESS: Se requiere acceso autorizado
     BLUEPRINTS:
       STATUS: Estado del plugin
       MODE: Modo mantenimiento
       MODE_HELP: Activa o desactiva el modo mantenimiento, en todo el sitio
       ALLOW_LOGIN: Permitir acceso
       ALLOW_LOGIN_HELP: Si deseas proporcionar un formulario de inicio de sesión para que los usuarios autorizados puedan acceder al sitio
-      ACCESS: Acceso
+      ACCESS: Accesso
```

Comment:
This is probably a display bug from Github? First I don't remember changing this and I don't understand why I see `Accesso` here and `Acceso` when trying to modify the file to the good translation